scout/server/blueprints/variants/templates/variants/cancer-variants.html

```diff
@@ -83,17 +83,18 @@
             </td>
             <td>{{ gene_cell(variant) }}</td>
             <td>
+
               <a target="_blank" href="{{ url_for('variant.cancer_variant', institute_id=institute._id, case_name=case.display_name,
                                   variant_id=variant._id, cancer='yes') }}">
                 {% if variant.first_rep_gene.hgvs_identifier %}
-                  <div>{{ variant.first_rep_gene.hgvs_identifier }}</div>
-                  <div>{{ (variant.first_rep_gene.hgvsp_identifier or '') |url_decode }}</div>
+                  <div>{{ variant.first_rep_gene.hgvs_identifier|truncate(30, True) }}</div>
```

Comment:
This is better than current main on layout, but we should ideally have like a pretty print for these as well, e.g. giving coord and ins and/or del and the number of bp instead of the exact sequence of each if it goes over an annoying number of chars in total.

Reply:
Ideally it would be nice to have a common way for cancer and non-cancer variants (the HGVS of these are handled in an easier way). Here it is complicated by the first and second representative gene which can be missing also. Let's merge and revisit if we touch this code in the future